R/init.R

```diff
@@ -1,45 +1,47 @@
 #' Create Rhino application
 #'
-#' Generates the file structure of a Rhino application.
-#' Can be used to start a fresh project or to migrate an existing Shiny application
-#' created without Rhino.
+#' Generates the file structure of a Rhino application. Can be used to start a fresh project or to
+#' migrate an existing Shiny application created without Rhino.
 #'
-#' The recommended steps for migrating an existing Shiny application to Rhino:
-#' 1. Put all app files in the `app` directory,
-#' so that it can be run with `shiny::shinyAppDir("app")` (assuming all dependencies are installed).
-#' 2. If you have a list of dependencies in form of `library()` calls,
-#' put them in the `dependencies.R` file.
-#' If this file does not exist, Rhino will generate it based on `renv::dependencies("app")`.
-#' 3. If your project uses `{renv}`, put `renv.lock` and `renv` directory in the project root.
-#' Rhino will try to only add the necessary dependencies to your lockfile.
-#' 4. Run `rhino::init()` in the project root.
+#' The recommended steps for migrating an existing Shiny application to Rhino: 1. Put all app files
+#' in the `app` directory, so that it can be run with `shiny::shinyAppDir("app")` (assuming all
+#' dependencies are installed). 2. If you have a list of dependencies in form of `library()` calls,
+#' put them in the `dependencies.R` file. If this file does not exist, Rhino will generate it based
+#' on `renv::dependencies("app")`. 3. If your project uses `{renv}`, put `renv.lock` and `renv`
+#' directory in the project root. Rhino will try to only add the necessary dependencies to your
+#' lockfile. 4. Run `rhino::init()` in the project root.
```

Comment:
Each bullet point needs to be on a new line for this Markdown list to be rendered correctly. Was it a result of automatic restyling (styler)?

I'd like to drop a couple of general notes here:
1. Please avoid changes which are irrelevant to the goal of the PR (restyling in particular). This way:
    1. The diffs are smaller and more meaningful; the changes are easier to review.
    2. There is a lower risk of merge conflicts with other developers and they are easier to resolve when they occur.
2. You are always welcome to create a separate PR with such changes, even if there is no issue/task for it. If you think it would make our code better - go for it!
2. Ideally we want the code style to be consistent across the whole project. When in doubt, stick to the style of the surrounding code (function, file, directory).

Reply:
Yes, this is automatic styling. I should have noticed this either way. The original styling is very readable and understandable, and I made it worse by turning them into one big block.

I agree that these changes are irrelevant to the goal and may cause conflicts later on.  A very good reminder for me.